### PR TITLE
added Homepage and footers

### PR DESCRIPTION
**Pull request by omjadhav85**

### CODE REVIEW

**Comment by MayurChawla** [comment]
you are using good css practises, appreciation for that. Seems Nice.

File: styles/homepage.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+.category:hover .category-img {
+  filter: blur(3px);
+}
+
+.category:hover .category-name {
+  color: var(--primary-color);
+}
+
+.category-img {
+  width: 300px;
```
Comment (dubeyaditya29) [inline styles/homepage.css]: suggestion : rems can be used for >10px

**Review by dubeyaditya29 — COMMENTED** [review]
Code looks clean good to merge!

**Comment by omjadhav85** [comment]
Thanks Mayur and Aditya for the review!

**Pull request merged**